docker-compose.yml

```
@@ -14,6 +14,7 @@ services:
       - openwisp_static:/opt/openwisp/static
       - openwisp_media:/opt/openwisp/media
       - influxdb_data:/var/lib/influxdb
+      - ./customize/configuration/custom_django_settings.py:/opt/openwisp/openwisp/custom_settings.py:ro
```

```suggestion
      - ./customize/configuration:/opt/openwisp/openwisp/configuration:ro
```

Alterantively we could have something like:

/configuration/theme
/configuration/django

or 

/customization/theme
/customization/django